lib/modules/versioning/hermit/index.ts

```diff
@@ -0,0 +1,233 @@
+import { regEx } from '../../../util/regex';
+import { RegExpVersion, RegExpVersioningApi } from '../regex';
+import type { VersioningApiConstructor } from '../types';
+
+export class HermitVersioning extends RegExpVersioningApi {
```

Comment:
This is the first versioning class we have overriding regex versioning. Not an issue, just interesting 😄